include/gatekeeper_gk.h

```diff
@@ -196,6 +197,13 @@ struct gk_config {
 	/* Time for logging the basic measurements in ms. */
 	unsigned int       basic_measurement_logging_ms;
 
+	/*
+	 * When the flow hash table is full, Gatekeeper will
+	 * enable the insertion again only after cleaning up
+	 * a number of expired flow entries.
+	 */
+	unsigned int       scan_del_thresh;
```

Comment:
Let's place the new parameter after `flow_table_scan_iter` since they are used together.